FixMyBerlin/fixmy.frontend
# Reports/fix zoom behavior

This is an attempt to fix some of the zoom behavior in the reports app. 

- Map doesn't zoom out anymore when opening a detail panel from a zoom point that's further in than the standard zoom for the detail view
- Map shifts to an offset so that currently selected report is in the middle of the map view even when the detail panel is open
- The position of the "add new report" / "more info" button on the overview map was fixed to be in the center of the viewport and animate on opening/closing the detail panel
- The preview image in the map popup was restored, it was invisible because it was set as the background image on an anchor element without further content.

**There's a still to be resolved issue where sometimes markers and marker clusters are visible for the same reports. I haven't been able to track that down yet.**

Fixes #611

## How Has This Been Tested?

- Navigate to overview map, observe map zoom level and marker position when opening detail pane
- Open map using deep link, observe the same

## Checklist:
